test/DynamoCoreWpfTests/GraphViewTests.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+using Dynamo.Graph.Workspaces;
+using Dynamo.Models;
+using Dynamo.Visualization;
+using Dynamo.Wpf.Rendering;
+using DynamoCoreWpfTests.Utility;
+
+namespace DynamoCoreWpfTests
+{
+
+    /// <summary>
+    /// This class should not be used for DynamoView unit tests
```

Comment:
maybe add what it should be used for 😄 ?

Reply:
yup, I can add